samples/aws/rke2-sample.yaml

```diff
@@ -66,7 +66,7 @@ spec:
   template:
     spec:
       version: v1.25.3
-      clusterName: my-cluster
+      clusterName: ${CABPR_CLUSTER_NAME}
```

Comment:
`CLUSTER_NAME` should be used ideally.

Reply:
Changed `CABPR_CLUSTER_NAME` to `CLUSTER_NAME`